README.md

```diff
@@ -255,6 +255,21 @@ Adding your own asset transformation functions is fair game too. In
 fact, it's encouraged. Let's say you need to serve all assets from a
 Content Delivery Network ...
 
+#### Serving bundles under a custom URL prefix
```

Comment:
Hah! I just put it close to the optimizations bit, didn't actually read it :D will fix